templates/zerver/app/index.html

```diff
@@ -109,6 +109,7 @@
 <div id="manage_streams_container"></div>
 <div id="manage_groups_container"></div>
 <div id="drafts_table"></div>
+<div id="scheduled_messages"></div>
```

Comment:
Looking at naming of adjacent stuff, let's call this something like `scheduled_messages_overlay_container`; I think the `settings` one just below this is named the best of the lot. 

@sahil839 FYI in case you agree and want to rename the manage streams/groups ones.

Reply:
Done.